This patch works with mocha.
Updated Author: Utkarsh Gupta <guptautkarsh2102@gmail.com>
---
--- a/test.js
+++ b/test.js
@@ -1,18 +1,14 @@
-import test from 'ava';
-import stripAnsi from '.';
+var m = require('.');
+var assert = require('assert');
 
-test('strip color from string', t => {
-	t.is(stripAnsi('\u001B[0m\u001B[4m\u001B[42m\u001B[31mfoo\u001B[39m\u001B[49m\u001B[24mfoo\u001B[0m'), 'foofoo');
+test('strip color from string', function() {
+    assert(m('\u001B[0m\u001B[4m\u001B[42m\u001B[31mfoo\u001B[39m\u001B[49m\u001B[24mfoo\u001B[0m') === 'foofoo');
 });
 
-test('strip color from ls command', t => {
-	t.is(stripAnsi('\u001B[00;38;5;244m\u001B[m\u001B[00;38;5;33mfoo\u001B[0m'), 'foo');
+test('strip color from ls command', function() {
+    assert(m('\u001B[00;38;5;244m\u001B[m\u001B[00;38;5;33mfoo\u001B[0m') === 'foo');
 });
 
-test('strip reset;setfg;setbg;italics;strike;underline sequence from string', t => {
-	t.is(stripAnsi('\u001B[0;33;49;3;9;4mbar\u001B[0m'), 'bar');
-});
-
-test('strip link from terminal link', t => {
-	t.is(stripAnsi('\u001B]8;;https://github.com\u0007click\u001B]8;;\u0007'), 'click');
+test('strip reset;setfg;setbg;italics;strike;underline sequence from string', function() {
+    assert(m('\u001B[0;33;49;3;9;4mbar\u001B[0m') === 'bar');
 });
